verification/curator-service/api/src/clients/aws-lambda-client.ts

```diff
@@ -83,15 +83,30 @@ export default class AwsLambdaClient {
 
     /**
      * Invoke retrieval function lambda synchronously, returning its output.
+     *
+     * @param sourceId - ID of the source to be retrieved.
+     * @param parseStartDateString - YYYY-MM-DD on which to begin parsing (inclusive).
+     * @param parseEndDateString - YYYY-MM-DD on which to end parsing (inclusive).
      */
-    invokeRetrieval = async (sourceId: string): Promise<RetrievalPayload> => {
+    invokeRetrieval = async (
+        sourceId: string,
+        parseStartDateString?: string,
```

Comment:
nit but as the start and end dates are to be used together I'd have bundled them in a struct to pass them as parameters

Reply:
Done.